source/features/file-finder-buffer.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'dom-chef';
+import features from '../libs/features';
+import select from 'select-dom';
+
+const pjaxStartHandler = ((event: CustomEvent): void => {
+	const destinationURL: string = event.detail?.url || '';
+	if (destinationURL.split('/')[5] === 'find') {
+		const hiddenInput = (
+			<input
+				type="text"
+				className="sr-only"
+				id="rgh-file-finder-buffer"
+			/>
+		);
+		document.body.prepend(hiddenInput);
+		hiddenInput.focus();
+	}
+}) as EventListener; // Explicit type cast. See https://github.com/microsoft/TypeScript/issues/28357#issuecomment-436484705
```

Comment:
This is because `pjax:start` is of type `CustomEvent`, and adding a listener for a custom event makes `window.addEventListener('pjax:start', pjaxStartHandler)` throw the following error: `No overload matches this call.` Searched it up and found the linked issue. Another workaround is to cast type of `event` as `any`.

Reply:
I think you just need to add both new events to https://github.com/sindresorhus/refined-github/blob/825f4fd51efaef62a5f131a83b2c7a2cc14116ee/source/globals.d.ts#L33

And it will work thanks to this generic type https://github.com/microsoft/TypeScript/blob/177713ef45f7714c91af79246b8b5e2b6bd59128/lib/lib.dom.d.ts#L5546

Reply:
Or maybe add this to `global.d.ts`

```ts
interface EventSourceEventMap {
	'pjax:start': CustomEvent
	etc
}
```